nested/track.json

```diff
@@ -0,0 +1,52 @@
+{
+	"short-description": "Nested query benchmark using up to 11,203,029 questions from StackOverflow",
+	"description": "This test indexes and queries up to 11,203,0299 questions from StackOverflow (total 3.64 GB json)",
+	"data-url": "http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora/nested",
+	"indices": [{
+		"name": "sonested",
+		"types": [{
+			"name": "question",
+			"mapping": "mappings.json",
+			"documents": "documents.json.bz2",
+			"document-count": 11203029,
+			"compressed-bytes": 695293381,
+			"uncompressed-bytes": 3637747670
+		}]
+	}],
+	"operations": [{
+		"name": "index-append",
+		"operation-type": "index",
+		"bulk-size": 500
```

Comment:
maybe we should have a larger bulk size since docs are tiny? For instance geonames seems to be using 5k as a mulk size.